lib/perl/Genome/Model/Build/ReferenceSequence/Converter.pm

```diff
@@ -57,11 +57,20 @@ sub __errors__ {
     return @errors;
 }
 
+sub exists_for_references {
+    my $class = shift;
+    my $source_reference = shift;
+    my $destination_reference = shift;
+
+    my @ref = $class->_faster_get(source_reference_build => $source_reference, destination_reference_build => $destination_reference);
+    return @ref;
```

Comment:
I almost wrote
```perl
return @ref? 1 : 0;
```
or
```perl
return !!@ref;
```

That would stop someone from abusing the return value.  I could change it if that would be better.